macSetup.sh

```diff
@@ -0,0 +1,3 @@
+# Add an alias for the SCE cli in .zshrc file
+echo "./sce.sh completion" >> ~/.bashrc
```

Comment:
do we need this? cant the user just copy paste this line in their own terminal

Reply:
+1. This file is unneeded